### PR TITLE
feat(cloud-service): change page store from `vuex` to `pinia`

### PR DESCRIPTION
**Pull request by yuda110**
### To Reviewers
- [ ] Skip (`style`, `chore` ONLY)
- [ ] Not that difficult

### Type of Change
- [x] New feature
- [ ] Bug fixes
- [ ] Feature improvement
- [x] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Checklist
- [x] `Error / Warning / Lint / Type`

### Description
SSIA

### CODE REVIEW

File: src/services/asset-inventory/cloud-service/CloudServicePage.vue [hunk 1]
```diff
@@ -173,24 +173,23 @@ export default {
...
-            searchFilters: computed<ConsoleFilter[]>(() => assetInventoryStore.state.cloudService.searchFilters),
-            selectedCategories: computed<CloudServiceCategory[]>(() => assetInventoryStore.getters['cloudService/selectedCategories']),
-            selectedRegions: computed<string[]>(() => assetInventoryStore.getters['cloudService/selectedRegions']),
-            allFilters: computed<ConsoleFilter[]>(() => assetInventoryStore.getters['cloudService/allFilters']),
+            title: computed<string>(() => {
+                if (!isEmpty(storeState.providers[cloudServicePageState.selectedProvider])) {
+                    return storeState.providers[cloudServicePageState.selectedProvider].name;
+                }
+                return cloudServicePageState.selectedProvider;
+            }),
```
Comment (WANZARGEN) [inline src/services/asset-inventory/cloud-service/CloudServicePage.vue:181]: question: I wonder why was this added 😮

File: src/services/asset-inventory/cloud-service/CloudServicePage.vue [hunk 2]
```diff
@@ -260,11 +255,11 @@ export default {
...
-            assetInventoryStore.dispatch('cloudService/setSelectedProvider', urlQueryValue.provider);
-            assetInventoryStore.dispatch('cloudService/setSelectedRegions', urlQueryValue.region);
-            assetInventoryStore.dispatch('cloudService/setSelectedCategories', urlQueryValue.service);
-            assetInventoryStore.dispatch('cloudService/setPeriod', urlQueryValue.period);
-            assetInventoryStore.dispatch('cloudService/setSearchFilters', searchQueryHelper.filters);
+            cloudServicePageStore.setSelectedProvider(urlQueryValue.provider);
+            cloudServicePageStore.setSelectedRegionsToFilters(urlQueryValue.region);
+            cloudServicePageStore.setSelectedCategoriesToFilters(urlQueryValue.service);
+            cloudServicePageState.period = urlQueryValue.period;
+            cloudServicePageState.searchFilters = searchQueryHelper.filters;
```
Comment (WANZARGEN) [inline src/services/asset-inventory/cloud-service/CloudServicePage.vue:262]: praise: much better to read :)

**Review by WANZARGEN — APPROVED** [review]
LGTM!

**Review by sulmoJ — APPROVED** [review]
Gooooood!

File: src/services/asset-inventory/cloud-service/CloudServicePage.vue (hunk 1 above)
Comment (yuda110) [reply src/services/asset-inventory/cloud-service/CloudServicePage.vue:181]: Original code was this (in template!)
I thought it was too long and unreadable, so change it to computed variable.

```
:title="storeState.providers[selectedProvider] ? storeState.providers[selectedProvider].name : selectedProvider"
```

Comment (WANZARGEN) [reply src/services/asset-inventory/cloud-service/CloudServicePage.vue:181]: Oh, thanx for explanation 😄
